Source/DafnyLanguageServer/StandaloneServerCli.cs

```diff
@@ -0,0 +1,56 @@
+﻿using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Logging;
+using OmniSharp.Extensions.LanguageServer.Server;
+using Serilog;
+using System;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Dafny.LanguageServer.Language;
+using Microsoft.Dafny.LanguageServer.Workspace;
+using Microsoft.Extensions.DependencyInjection;
+using OmniSharpLanguageServer = OmniSharp.Extensions.LanguageServer.Server.LanguageServer;
+
+namespace Microsoft.Dafny.LanguageServer {
+  public class Program {
+    public static async Task Main(string[] args) {
```

Comment:
Did `DafnyLanguageServer` not already have a Main method somewhere? If not, why do we need this if we can run `dafny server` instead?

Reply:
Never mind, spotted it after posting the review: https://github.com/dafny-lang/dafny/pull/3088/files#diff-c18f4e183c062f2e8deb2355f01745c4050117bbb0b02838b1567406ec544c1dL15 :)